magento/magento2
# Enable/Disable DB query logging commands

### Description

Added console commands for enabling/disabling the db query logging.

### Fixed Issues (if relevant)

1. magento/magento2#9200: Create new CLI command: Enable dev settings, i.e DB logging, Profiler, Template Hints

### Manual testing scenarios

1. Run bin/magento dev:query-log:enable command.
2. Clear configuration cache.
3. Make any request to the Magento site and verify that the DB query log is being written.
4. Run bin/magento dev:query-log:disable command.
5. Clear configuration cache
6. Make any request to the site and verify that the DB query log have no new log data.
